tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -1,12 +1,11 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
... Honestly not sure how that change happened. Maybe a rebase at some point? I will change this back.